integrated_channels/canvas/admin/__init__.py

```diff
@@ -62,6 +66,38 @@ def customer_oauth_authorization_url(self, obj):
         else:
             return None
 
+    def force_content_metadata_transmission(self, request, obj):
+        """
+        Updates the modified time of the customer record to retransmit courses metadata
+        and redirects to configuration view with success or error message.
+        """
+        try:
+            obj.enterprise_customer.save()
+            messages.success(
+                request,
+                "The canvas enterprise customer content metadata "
+                "“<CanvasEnterpriseCustomerConfiguration for Enterprise {enterprise_name}>” "
+                "was updated successfully.".format(
```

Comment:
(tiny nit) Generally we've moved from python's string `format` to `f-strings` when interpolating